### PR TITLE
[DE] Add `Help_Centre\Account`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
This PR itself is ready for review. My own translation of line 218 feels a bit odd. If someone has a better idea, please suggest it!

However, a link needs to be changed in the article `People\The_Team\Account_support_team`. So, there are the following tasks waiting for completion:
- [x] Requires #6584 
- [x] Change link in `Account_support_team` as soon as the PR above has been merged (The CI failure is related to that and will be cleared with this task anyway.)

Luckily, I noticed the recent change ([#6618](https://github.com/ppy/osu-wiki/pull/6618)) in this article regarding the editing of country flags, because I was in the midst of the translation.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Help_Centre/Account/de.md [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+
+## Andere Probleme
+
+### Die Flagge/das Land auf meinem Profil ist falsch!
+
+**Leider können wir das nicht für dich ändern.**
+
+Das mit deinem osu!-Profil verknüpfte Land basiert auf der geographischen Lage der Verbindung, die du benutzt hast, als du deinen Account registriert hast. Das Land wird deinem Account dauerhaft zugewiesen.
+
+In der Vergangenheit haben wir unseren Spieler die Möglichkeit gegeben, die Flagge ihres Accounts ändern zu können, sofern sie Beweise für ihren Aufenthalt liefern konnten. Nach etlichen Fällen von Missbrauch durch skrupellose Spieler hatten wir keine andere Wahl als dieses Angebot einzustellen.
```
Comment (respektive) [inline wiki/Help_Centre/Account/de.md:198]:
```suggestion
In der Vergangenheit haben wir unseren Spielern die Möglichkeit gegeben, die Flagge ihres Accounts ändern zu können, sofern sie Beweise für ihren Aufenthalt liefern konnten. Nach etlichen Fällen von Missbrauch durch skrupellose Spieler hatten wir keine andere Wahl als dieses Angebot einzustellen.
```

File: wiki/Help_Centre/Account/de.md [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+
+**So lange du auf deinem eigenem Account spielst**, kannst du osu! überall spielen!
+
+Solltest du in einem öffentlichen Ort spielen, achte auf die Sicherheit deines Accounts. Geteilte Computer sind oft Ziele von Angriffen mit Schadsoftware.
+
+### Kann ich eine Liste aller Maps bekommen, die ich gespielt oder heruntergeladen habe?
+
+**Du kannst über die Webseite alle Maps einsehen, die du gespielt hast.**
+
+**Wenn du ein aktives Supporter-Tag besitzt**, kannst du alle Maps sehen, auf denen du ein Score erzielt hast. Dazu musst du auf die [beatmap-auflistung](https://osu.ppy.sh/beatmapsets) gehen, auf `Mehr Suchoptionen` klicken und dann den Filter `Gespielt` aktivieren. Alternativ kannst du auch [die gefilterte Liste an Beatmaps](https://osu.ppy.sh/beatmapsets?played=played) direkt abrufen.
```
Comment (respektive) [inline wiki/Help_Centre/Account/de.md:242]:
```suggestion
**Wenn du ein aktives Supporter-Tag besitzt**, kannst du alle Maps sehen, auf denen du einen Score erzielt hast. Dazu musst du auf die [beatmap-auflistung](https://osu.ppy.sh/beatmapsets) gehen, auf `Mehr Suchoptionen` klicken und dann den Filter `Gespielt` aktivieren. Alternativ kannst du auch [die gefilterte Liste an Beatmaps](https://osu.ppy.sh/beatmapsets?played=played) direkt abrufen.
```

File: wiki/Help_Centre/Account/de.md [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+
+Leider können wir nicht speichern, welche Beatmaps ein Spieler heruntergeladen hat. Das heißt, es ist unmöglich davon eine Auflistung davon zu erhalten. Das nächste, wie du an das rankommen würdest, wäre die Benutzung des zuvor erwähnten Filters, um zumindest an die vorher gespielten Maps zu gelangen.
```
Comment (respektive) [inline wiki/Help_Centre/Account/de.md:244]:
```suggestion
Leider speichern wir nicht, welche Beatmaps ein Spieler heruntergeladen hat. Das heißt, es ist unmöglich davon eine Auflistung zu erhalten. Das nächste, wie du an das rankommen würdest, wäre die Benutzung des zuvor erwähnten Filters, um zumindest an die vorher gespielten Maps zu gelangen.
```

**Review by respektive — CHANGES_REQUESTED** [review]
just some small grammar changes/typo fixes, looks pretty good otherwise 👍

**Review by Walavouchey — APPROVED** [review]
(force-pushed to fix off-by-one error)

**Comment by The-Last-Cookie** [comment]
Thanks for applying the changes 👍

**Pull request merged**